src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,13 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    private Random random = new Random();
+
+    protected String getRandomColor() {
```

Comment:
Should it be `protected` or, maybe, `public`? Protected methods cannot be used in other non-subclass packages.